.github/workflows/test-e2e-create.yml

```diff
@@ -0,0 +1,57 @@
+name: E2E test - create command
+on:
+  push:
+    branches:
+      - main
+      - 'release-**'
+  pull_request:
+    branches:
+      - main
+      - 'release-**'
+jobs:
+  test-create-cmd:
+    runs-on: ubuntu-latest
+    env:
+      K3D_VERSION: v5.4.7
+      MODULE_TEMPLATE_VERSION: 1.0.0
+      OCI_REPOSITORY_URL: http://k3d-oci.localhost:5001
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Go setup
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: 'go.mod'
+          cache-dependency-path: 'go.sum'
+      - name: Build
+        run: |
+          make build-linux
+          chmod +x ./bin/modulectl-linux
+          ls -la ./bin
+          mv ./bin/modulectl-linux /usr/local/bin/modulectl
+        timeout-minutes: 5
+      - name: Checkout template-operator
+        uses: actions/checkout@v4
+        with:
+          repository: kyma-project/template-operator
+          path: ./template-operator/
+      - name: Copy template-operator to e2e testdata folder
+        run: |
+          mkdir -p ./tests/e2e/create/testdata/template-operator/
+          cp -r ./../template-operator/ ./tests/e2e/create/testdata/template-operator/
```

Comment:
Thanks, good find. I was unsure where to keep the testdata, for local testing then.

Reply:
I think here we should not checkout template-operator repo.

The modulectl create command should not rely on a specific module repo, the required content are default-cr.yaml and raw-manifest.yaml, all of them can be downloaded in template-operator release page.

It's also better we test against the latest released manifests, to prevent inconsistency brings by template-operator due to checkout main branch.

I suggest refactoring this step to download latest release assets as provided input.